### DISCUSSION TITLE
Remove username from the 500 lines

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#16807] [General]
Could it make sense that mentions in Mastodon posts don't count towards the 500 character limit?

**Comment by Gargron** [comment]
Domains in mentions don’t count, but I see value in limiting the amount of people you can mention in a single post.

**Reply by linux-lukas** [reply]
Yes, to limit the number of account can mention in a posting is OK.

Could Content Warnings (CWs) also be excluded from the 500 characters?
